packages/main/test/pages/Toast.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta charset="utf-8">
+	<title>ui5-toast</title>
+
+	<script src="../../webcomponentsjs/webcomponents-loader.js"></script>
+	<script src="../../resources/bundle.esm.js" type="module"></script>
+	<script nomodule src="../../resources/bundle.es5.js"></script>
+	<script>delete Document.prototype.adoptedStyleSheets;</script>
+	<script src="modules/Toast.js"></script>
+
+	<style>
+		ui5-button, h2 {
+			margin: 1rem;
+		}
+	</style>
+</head>
+
+<body onload="onload()">
+
+	<h2>Toast ( ui5-toast )</h2>
+
+	<!--TopStart-->
+	<ui5-button class="wcBtnOpenToastTS">Show TopStart Toast</ui5-button>
```

Comment:
Just a hint, if you use IDs instead of classes, then you can do this in the script:
```js
btnShowToastId.addEventListener('click', function () {
	myToastId.show();
});
```